File: ui/src/main/java/com/peakmain/ui/utils/launcher/task/MainTask.java
package com.peakmain.ui.utils.launcher.task;

/**
 * author ：Peakmain
 * createTime：2020/12/10
 * mail:2726449200@qq.com
 * describe：主任务
 */
public abstract class MainTask extends Task{
    @Override
    public boolean runOnMainThread() {
        return true;
    }
}

File: ui/src/main/java/com/peakmain/ui/utils/launcher/task/MainTask.kt
package com.peakmain.ui.utils.launcher.task

/**
 * author ：Peakmain
 * createTime：2020/12/10
 * mail:2726449200@qq.com
 * describe：主任务
 */
abstract class MainTask : Task() {
    override fun runOnMainThread(): Boolean {
        return true
    }
}